<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CUserProfile</class>
 <widget class="QDialog" name="CUserProfile">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>376</width>
    <height>149</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>User profile</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>User:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxType">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>100</y>
     <width>261</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Engineering user (Upload recipes only)</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Production user (Upload + Enable recipes)</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Administrator (All rights)</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>46</width>
     <height>14</height>
    </rect>
   </property>
   <property name="text">
    <string>Rights:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditUserName">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>18</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonOk">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>10</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Ok</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonCancel">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>40</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditPassword">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>58</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_Password">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Password:</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>lineEditUserName</tabstop>
  <tabstop>lineEditPassword</tabstop>
  <tabstop>comboBoxType</tabstop>
  <tabstop>pushButtonOk</tabstop>
  <tabstop>pushButtonCancel</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
